settings.gradle

```diff
@@ -43,7 +43,8 @@ include 'fiat-api',
         'fiat-google-groups',
         'fiat-ldap',
         'fiat-roles',
-        'fiat-web'
+        'fiat-web',
+        'fiat-ldap-v2'
```

Comment:
Pretty minor, but I would keep this alphabetized by inserting this line beneath `'fiat-ldap',`